### PR TITLE
Bug Fix: Block order doesn't change dance

### PR DESCRIPTION
**Pull request by epeach**
Addresses: https://github.com/code-dot-org/dance-party/issues/43

* Sorts cues (with test)

* After discussion with JoshL, adds p5.dance.interpreted.js to dance-party

### CODE REVIEW

File: src/p5.dance.js [hunk 1]
```diff
@@ -111,7 +110,11 @@ module.exports = class DanceParty {
   }
 
   addCues(timestamps) {
-    this.world.cues = timestamps;
+    // Sort and de-duplicate
+    const numSort = (a,b) => a - b;
```
Comment (joshlory) [inline src/p5.dance.js]: Also need to de-duplicate?

File: test/unit/cuesTest.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const test = require('tape');
+const DanceParty = require('../../src/p5.dance');
+
+const createDanceAPI = () => {
```
Comment (joshlory) [inline test/unit/cuesTest.js]: Nit: we're starting to duplicate this a number of places, I wonder if its worth extracting to a test/helpers util?

(Non-blocking, we could address this in a follow up PR.)

File: test/unit/cuesTest.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  nativeAPI.play({
+    bpm: 120,
+  });
+
+  //Initial Value
+  t.deepEqual(nativeAPI.world.cues.measures, []);
+  t.deepEqual(nativeAPI.world.cues.seconds, []);
+
+  let cues = {measures: [0, 1, 4, 2, 0, 8, 3, 3, 3], seconds: [0, 1, 4, 2, 0, 8, 3, 3, 3]};
+  let sortedCues = [0, 0, 1, 2, 3, 3, 3, 4, 8];
```
Comment (joshlory) [inline test/unit/cuesTest.js]: Should be `[0, 1, 2, 3, 4, 8];` if we de-duplicate?

File: src/p5.dance.js (hunk 1 above)
Comment (epeach) [reply src/p5.dance.js]: Whoops! Leftover comment from when I thought we'd need to de-duplicate. The while loop in updateEvents means that duplicates are already handled as long as they are sorted. I'll update that comment.

Comment (joshlory) [reply src/p5.dance.js]: Don't we want to de-duplicate?  I guess it's fine for the `while` loop to take care of it... but sort of prefer to sanitize the data.

**Pull request merged**